### PR TITLE
use `run_shell_cmd` in custom easyblock for OpenFOAM

### PR DESCRIPTION
**Pull request by branfosj**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by branfosj** [comment]
Test report by @branfosj
#### Overview of tested easyconfigs (in order)
 * **SUCCESS** _OpenFOAM-v2312-foss-2023a.eb_ 
 * **SUCCESS** _OpenFOAM-11-foss-2023a.eb_ 
 * **FAIL (build issue)** _OpenFOAM-Extend-5.0-20230821-foss-2022a.eb_ (partial log available at https://gist.github.com/branfosj/1066b9b5e0cc09ef405d28d515b0d558)
 * **SUCCESS** _ParaView-5.9.1-foss-2021a-mpi.eb_ 
 * **SUCCESS** _CGAL-5.2-gompi-2020b.eb_ 
 * **SUCCESS** _ParaView-5.8.1-foss-2020b-mpi.eb_ 
 * **SUCCESS** _libcerf-1.17-GCCcore-10.3.0.eb_ 
 * **SUCCESS** _Lua-5.4.3-GCCcore-10.3.0.eb_ 
 * **FAIL (build issue)** _libgd-2.3.0-GCCcore-10.2.0.eb_ (partial log available at https://gist.github.com/branfosj/a67e121db7ce3a2abf78d52660b26d80)
 * **FAIL (build issue)** _gnuplot-5.4.2-GCCcore-10.3.0.eb_ (partial log available at https://gist.github.com/branfosj/48186142288554bf69c51a457116757c)
 * **FAIL (build issue)** _OpenFOAM-v2106-foss-2021a.eb_ (partial log available at https://gist.github.com/branfosj/5e48500bec1acc0f04967c69194bf32d)
 * **SUCCESS** _libcerf-1.14-GCCcore-10.2.0.eb_ 
 * **SUCCESS** _Lua-5.4.2-GCCcore-10.2.0.eb_ 
 * **FAIL (build issue)** _gnuplot-5.4.1-GCCcore-10.2.0.eb_ (partial log available at https://gist.github.com/branfosj/94b5760d6dada2313df7898bcdd19d13)
 * **FAIL (build issue)** _OpenFOAM-8-foss-2020b.eb_ (partial log available at https://gist.github.com/branfosj/f3a45c7699994ab7fb25df46b0311cb8)

Build succeeded for 9 out of 15 (5 easyconfigs in total)
bear-pg0105u03a - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/34471d21fc399845ae154c2f5c5694d0 for a full test report.

File: easybuild/easyblocks/o/openfoam.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ##
-# Copyright 2009-2024 Ghent University
+# Copyright 2009-2023 Ghent University
```
Comment (branfosj) [inline easybuild/easyblocks/o/openfoam.py:2]:
```suggestion
# Copyright 2009-2024 Ghent University
```

**Comment by branfosj** [comment]
Test report by @branfosj
#### Overview of tested easyconfigs (in order)
 * **SUCCESS** _OpenFOAM-Extend-5.0-20230821-foss-2022a.eb_ 
 * **SUCCESS** _OpenFOAM-v2106-foss-2021a.eb_ 
 * **SUCCESS** _OpenFOAM-v2312-foss-2023a.eb_ 
 * **SUCCESS** _OpenFOAM-8-foss-2020b.eb_ 
 * **SUCCESS** _OpenFOAM-11-foss-2023a.eb_ 

Build succeeded for 5 out of 5 (5 easyconfigs in total)
bear-pg0105u03a - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/66ad50f65e39359d4c21a459859df171 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
#### Overview of tested easyconfigs (in order)
 * **SUCCESS** _OpenFOAM-10-foss-2022a.eb_ 
 * **SUCCESS** _OpenFOAM-v2306-foss-2022b.eb_ 

Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3524.doduo.os - Linux RHEL 8.8, x86_64, AMD EPYC 7552 48-Core Processor, Python 3.6.8
See https://gist.github.com/boegel/afa4b34221c8a7f00aac60ba2ecbcda5 for a full test report.

**Pull request merged**